libs/libplatforminfo/Config.in

```diff
@@ -0,0 +1,33 @@
+menu "Configuration"
+	depends on PACKAGE_libplatforminfo
+
+config LIBPLATFORMINFO_CUSTOM
+	bool "Use custom definitions"
+	default n
+	help
+		Use custom compilation target to get
+		the definitions for the platform informations
+
+config LIBPLATFORMINFO_BOARD_NAME
+	string "Path to the board name information"
+	default "/tmp/sysinfo/board_name"
+	depends on LIBPLATFORMINFO_CUSTOM
```

Comment:
Maybe wrap the whole block into `if LIBPLATFORMINFO_CUSTOM` instead of adding depends for each symbol?

Reply:
fixed